provider/blob/Cargo.toml

```diff
@@ -48,6 +48,8 @@ icu_testdata = { version = "0.5", path = "../testdata", features = ["static"] }
 path = "src/lib.rs"
 
 [features]
+default = ["sync"]
```

Comment:
Nit: You don't need a `"sync"` feature on this crate. You only need it on `icu_provider`.

Reply:
I think it's weird not to have it here. `BlobDataProvider` is the concrete type that clients will be using, and it's weird if that changes its `Sync`ness depending on the features of another crate.

Reply:
I think `BlobDataProvider` is not the _only_ crate like this, and it's reasonably normal for dependency crate features to be toggled this way. The type comes from `icu_provider`, people will be importing `icu_provider` (it's not a secret internal crate), they can enable the feature there directly.

The feature should go on BlobDataProvider if BlobDataProvider also does something special for it.